action/recipients.go

```diff
@@ -68,7 +68,10 @@ func (s *Action) RecipientsAdd(c *cli.Context) error {
 		}
 		keys = keys.UseableKeys()
 		if len(keys) < 1 {
-			return fmt.Errorf("no matching valid key found in keyring")
+			fmt.Println(color.CyanString("Warning: No matching valid key found. If the key is in your keyring you may need to trust it."))
```

Comment:
_"you may need to trust it"_
Technically the need to **validate** it  (by either 1/ trusting at `ultimate` level or 2/ signing it locally)  

cc @dominikschulz